### PR TITLE
Outstanding patches from the DAT hackathon

### PR DESCRIPTION
**Pull request by bgeltz**
These patches were necessary to perform the experiments on our most recent DAT.

### CODE REVIEW

File: integration/experiment/energy_efficiency/Makefile.mk [hunk 1]
```diff
@@ -36,6 +36,7 @@ EXTRA_DIST += integration/experiment/energy_efficiency/barrier_frequency_sweep.p
               integration/experiment/energy_efficiency/run_barrier_frequency_sweep_nekbone.py \
               integration/experiment/energy_efficiency/run_power_balancer_energy_dgemm.py \
               integration/experiment/energy_efficiency/run_power_balancer_energy_nasft.py \
+              integration/experiment/energy_efficiency/run_power_balancer_energy_hpcg_mkl.py \
```
Comment (dianarg) [inline integration/experiment/energy_efficiency/Makefile.mk]: this one is in the other PR; I renamed it to run_power_balancer_energy_hpcg.py now that there's only one version

**Review by dianarg — DISMISSED** [review]

File: integration/apps/nasft/nasft/npbparams.fi [hunk 2]
```diff
@@ -41,7 +41,7 @@
 !
         integer nx, ny, nz, maxdim, niter_default
         integer*8 np_min, ntdivnp
-        parameter (nx=2048, ny=1024, nz=1024, maxdim=2048)
+        parameter (nx=4096, ny=2048, nz=2048, maxdim=4096)
```
Comment (cmcantalupo) [inline integration/apps/nasft/nasft/npbparams.fi]: I'm working on a patch that will enable this to be selected, if this is merged then nas-ft will not run on mcfly

File: integration/experiment/common_args.py [hunk 3]
```diff
@@ -125,7 +125,7 @@ def add_cool_off_time(parser):
 
 def add_agent_list(parser):
     parser.add_argument('--agent-list', dest='agent_list',
-                        action='store', type=str, default=None,
+                        action='store', type=str, default='power_governor',
```
Comment (cmcantalupo) [inline integration/experiment/common_args.py]: I'm not sure we want this in dev.

File: integration/apps/nasft/nasft/npbparams.fi (hunk 2 above)
Comment (bgeltz) [reply integration/apps/nasft/nasft/npbparams.fi]: OK, I'll remove it from this stack.

File: integration/experiment/common_args.py (hunk 3 above)
Comment (bgeltz) [reply integration/experiment/common_args.py]: It looks like ```add_agent_list()``` is only used in power_sweep.py, but power_sweep.py already has an override for the defaults for agent_list.

I'm confused on what bug this was a workaround for?

**Review by dianarg — APPROVED** [review]

**Pull request merged**